src/Devices/PHProbe.cpp

```diff
@@ -28,6 +28,13 @@ PHProbe::PHProbe() {
   Serial1.print(F("C,1\r"));    // Reset pH stamp to continuous measurement: once per second
 }
 
+void PHProbe::onePointCalibration(double midpoint) {
+  const String PARTIAL_COMMAND = "Cal,mid,";
```

Comment:
We had a partial command in the earlier function (temperature?) because we had branching such that the same literal was used twice. Here we don't have branching so there is less reason to make an extra variable. We might just as well add it to the expression below as a literal.